canonical_mutation.cc

```diff
@@ -89,3 +90,81 @@ mutation canonical_mutation::to_mutation(schema_ptr s) const {
     }
     return m;
 }
+
+static sstring bytes_to_text(bytes_view bv) {
+    sstring ret(sstring::initialized_later(), bv.size());
+    std::copy_n(reinterpret_cast<const char*>(bv.data()), bv.size(), ret.data());
+    return ret;
+}
+
+std::ostream& operator<<(std::ostream& os, const canonical_mutation& cm) {
+    auto in = ser::as_input_stream(cm._data);
+    auto mv = ser::deserialize(in, boost::type<ser::canonical_mutation_view>());
+    column_mapping mapping = mv.mapping();
+    auto partition_view = mutation_partition_view::from_view(mv.partition());
+    fmt::print(os, "{{canonical_mutation: ");
+    fmt::print(os, "table_id {} schema_version {} ", mv.table_id(), mv.schema_version());
```

Comment:
Would it be helpful to also print the `keyspace.table_name`?

Reply:
Yes, in my use case I guessed the table from the column names, but it's better to print them.

Reply:
In fact the keyspace and table names are not available in the canonical mutation.